test/explorer.spec.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-dom/test-utils';
+import Explorer from '../src/scripts/components/explorer/Explorer';
+
+const moment = require('moment');
```

Comment:
lines 6-13:
I made an arbitrary style decision to just stick with ES6's `import` now instead of `require` so can you change these? The `import` will get converted to `require` w/ babel right now anyway, so might as well stick with one way of including external libraries.  I'll go ahead and change the other files (non Explorer) files.